frontend/packages/console-app/src/components/console-operator/ConsoleOperatorConfig.tsx

```diff
@@ -254,6 +260,20 @@ const ConsolePluginsList: React.FC<ConsolePluginsListType> = ({ obj }) => {
           )}
         />
       )}
+      <RequireCreatePermission model={ConsolePluginModel}>
```

Comment:
Due to the one-off nature of this page, I duplicated [the relevant `createLink` bits from `ListPage`](https://github.com/openshift/console/blob/7993fd9f652aac0c4c4f7f83e529a46e799b530c/frontend/public/components/factory/list-page.tsx#L266-L311).